ko/csql.rst

```diff
@@ -824,3 +824,26 @@ CSQL 인터프리터에서 작업 중인 데이터베이스 이름 및 호스트
     SINGLE IS OFF
     csql> ;singleline
     SINGLE IS OFF
+
+**새로운 csql 세션 연결(;Connect)**
+
+CSQL을 종료하지 않고 다른 사용자로 접속을 전환하기 위한 세션 명령어이다. 명령어 형식은 다음과 같다. 
+
+* user: 접속할 사용자 이름 
+* database: 접속할 데이터베이스 이름 (생략 시, 현재 접속한 데이터베이스)
+* host: 접속할 호스트 이름 (생략 시, localhost)::
```

Comment:
Here also need to fix the clone behind localhost

Reply:
Thank you. I applied it.